<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmployeSpWidget</class>
 <widget class="QWidget" name="EmployeSpWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>141</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item row="1" column="1">
    <widget class="QLabel" name="lSalaire">
     <property name="text">
      <string>Salaire</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QDoubleSpinBox" name="dsbSalaire">
     <property name="maximum">
      <double>9999.989999999999782</double>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="lConje">
     <property name="text">
      <string>Conje:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QSpinBox" name="sbConje">
     <property name="maximum">
      <number>99</number>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="lDateEmbau">
     <property name="text">
      <string>Date d'embauche:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QCheckBox" name="cbRetraite">
     <property name="text">
      <string>Retraite</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <layout class="QHBoxLayout" name="hlDateEmbau">
     <item>
      <widget class="QSpinBox" name="sbDay">
       <property name="maximum">
        <number>30</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="sbMonth"/>
     </item>
     <item>
      <widget class="QSpinBox" name="sbYear">
       <property name="maximum">
        <number>2017</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
